Client/src/AppRouter/AppRouter.jsx

```diff
@@ -0,0 +1,37 @@
+import { Switch, Route } from "react-router-dom";
+import { publicRoutes, privateRoutes } from "../router";
+import { useContext } from "react";
+import { PagesContext } from "../PagesProvider";
+
+export default function AppRouter() {
+    const {isAuthenticated} = useContext(PagesContext)
+
+    console.log(isAuthenticated)
+
+    return (
+        isAuthenticated
+        ?
+            <Switch>
```

Comment:
Can we use it like this?
(isAuthenticated ? publicRoutes : privateRoutes).map(route =>